kraken-build/tests/kraken_std/integration/cargo/test_cargo_private_registry.py

```diff
@@ -171,13 +171,13 @@ def publish_lib_and_build_app(repository: CargoRepositoryWithAuth, tempdir: Path
 @pytest.fixture(scope="session")
 def private_registry(docker_service_manager: DockerServiceManager) -> str:
     container = docker_service_manager.run(
-        "ghcr.io/d-e-s-o/cargo-http-registry:latest",
+        "ghcr.io/d-e-s-o/cargo-http-registry:sha-2edffd8",  # TODO(Tpt): hardcoded because latest docker images are broken
         [
             "/tmp/test-registry",
             "--addr",
             "0.0.0.0:35504",
         ],
-        ports=["35504:35504"],
+        ports=["35504"],
```

Comment:
We load the assigned port just below, there is no reason to hardcode the port imho